gym_snake/envs/snakeGameGym.py

```diff
@@ -41,6 +42,8 @@ def __init__(self, fps: int, use_pygame: bool = True):
 		self.snake = Snake(self.rows,self.cols)
 		self.fruit_pos = (0,0)
 		self.generate_fruit()
+		self.bomb_pos = (0,0)
```

Comment:
create state variable for whether or not we are using bombs. Then pass this conditional to game_over and directly below here to determine whether bomb is genderated